### PR TITLE
Graduate PriorityClass API to GA

### PR DESCRIPTION
**Pull request by bsalamat**
**What type of PR is this?**
/kind api-change

(Not sure if the api-change is the right kind for this PR)

**What this PR does / why we need it**:
Graduate PriorityClass API to GA.

**Which issue(s) this PR fixes**:

Fixes #

kubernetes/enhancements#564

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
The PriorityClass API has been promoted to `scheduling.k8s.io/v1` with no changes. The `scheduling.k8s.io/v1beta1` version is now deprecated and will stop being served by default in v1.17.
```

### CODE REVIEW

File: pkg/kubectl/describe/versioned/describe.go [hunk 1]
```diff
@@ -3699,7 +3700,7 @@ type PriorityClassDescriber struct {
 }
 
 func (s *PriorityClassDescriber) Describe(namespace, name string, describerSettings describe.DescriberSettings) (string, error) {
-	pc, err := s.Scheduling().PriorityClasses().Get(name, metav1.GetOptions{})
+	pc, err := s.SchedulingV1().PriorityClasses().Get(name, metav1.GetOptions{})
```
[the remark this answers is not in the document]
Comment (liggitt) [reply pkg/kubectl/describe/versioned/describe.go]: > what happen when we upgrade cluster with existing PriorityClass, e.g. v1beta1?

The server would convert it to v1 when requested via the v1 api endpoint

**Comment by liggitt** [comment]
I'll take this review. This is a good example of a "promote without changes" API PR, will be good as an example to start a checklist for that type of review.

**Comment by liggitt** [comment]
review scheduled for 1/19

**Comment by liggitt** [comment]
one question about an open feature request that would appear to involve API changes: https://github.com/kubernetes/kubernetes/issues/67671#issuecomment-465313719

**Review by bsalamat — COMMENTED** [review]
Thanks, @liggitt! I addressed your comments. PTAL.

File: pkg/apis/scheduling/install/install.go [hunk 2]
```diff
@@ -38,5 +38,6 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(v1.AddToScheme(scheme))
 	utilruntime.Must(v1beta1.AddToScheme(scheme))
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1.SchemeGroupVersion, v1beta1.SchemeGroupVersion, v1alpha1.SchemeGroupVersion))
+	// TODO(bsalamat): update this in 1.15 to use v1 API.
+	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion, v1alpha1.SchemeGroupVersion))
```
Comment (liggitt) [inline pkg/apis/scheduling/install/install.go]: include v1, but put v1beta1 first

Comment (bsalamat) [reply pkg/apis/scheduling/install/install.go]: Done. Thanks!

**Comment by liggitt** [comment]
as a follow-up, please update the API documentation comments on the v1beta1 types to indicate they are deprecated in favor of the v1 versions.

**Pull request merged**